mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.DeleteOrder = (req, res) => {
```

Comment:
The function name `exports.DeleteOrder` should be changed to `exports.removeOrder` to maintain consistency with the naming convention used in the routes.